app/javascript/article-form/components/__tests__/LoadingPreview.test.jsx

```diff
@@ -14,12 +14,12 @@ describe('<LoadingPreview />', () => {
   it('should render', () => {
     const { queryByTitle } = render(<LoadingPreview />);
 
-    expect(queryByTitle('Loading preview...')).toBeDefined();
+    expect(queryByTitle('Loading preview...')).toBeNull();
   });
 
   it('should render with cover image', () => {
     const { queryByTitle } = render(<LoadingPreview version="cover" />);
 
-    expect(queryByTitle('Loading preview...')).toBeDefined();
+    expect(queryByTitle('Loading preview...')).toBeNull();
```

Comment:
I'm not sure that the reading of this test makes sense to me. I thought we're trying to make sure that "Loading Preview..." shows up? but this change checks if it is null?